aci/data_source_aci_bfdmhifpol.go

```diff
@@ -0,0 +1,70 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciBfdMultihopInterfacePolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciBfdMultihopInterfacePolicyRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"tenant_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"admin_state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"detection_multiplier": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"min_receive_interval": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"min_transmit_interval": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"descr": {
```

Comment:
Update all other references.

```suggestion
			"description": {
```